.gitlab-ci.yml

```diff
@@ -1,6 +1,8 @@
 ---
 # -*- coding: utf-8 -*-
 
+image: python:3
+
```

Comment:
additionally to this I think you also need sphinx for this to work.

You can add it to the script list as e.g. `pip install sphinx`

Reply:
Oh sure, I missed that.